.github/workflows/codeql.yml

```diff
@@ -30,16 +30,16 @@ jobs:
 
       - name: Initialize CodeQL
         id: initialize
-        uses: github/codeql-action/init@83923549f688e42b34d0b90ee94725f7c30532fc # v3.28.9
+        uses: github/codeql-action/init@b2e6519679e446e7bb7c3466d70f13a6b5461fcd # v3.28.9
```

Comment:
For better readability, I'd keep the semver comment updated as well.

Reply:
Updated comment